### PR TITLE
Add documentation for deployment steps

### PR DESCRIPTION
**Pull request by matthewroach**
Add documentation for deployment of bundles for QA..

### CODE REVIEW

File: News Theme Development.md [hunk 1]
```diff
@@ -127,6 +129,33 @@ git checkout -b TMEDIA-[jira ticket num]-[brief description of feature]
 4. Make pull request using Github against the `canary` branch. Get approval for your PR on your feature branch.
 5. Merge the PR into the `canary` branch. At this point a release with the dist-tag of `canary` will be built automatically. This means that if you want to verify your changes in a deployed environment, you need to make sure you're using the `canary` dist-tag in whatever environment that is by setting the `BLOCK_DIST_TAG` environment variable in your environment file(s).
 
+### Deployment for QA
+
+In the development process this takes care of doing the work and getting your changes into the `canary` tag. In order to see your changes on a live environment for QA you will need to, use the Fusion deployer to either deploy a new bundle or duplicate an existing one - http://redirector.arcpublishing.com/deployments/fusion/
```
Comment (JackHowa) [inline News Theme Development.md]:
```suggestion
In the development process this takes care of doing the work and getting your changes into the `canary` tag. In order to see your changes on a live environment for QA you will need to use the Fusion deployer to either deploy a new bundle or duplicate an existing one - http://redirector.arcpublishing.com/deployments/fusion/
```

**Review by JackHowa — DISMISSED** [review]
minor comma

**Review by JackHowa — APPROVED** [review]

File: News Theme Development.md [hunk 2]
```diff
@@ -127,6 +129,33 @@ git checkout -b TMEDIA-[jira ticket num]-[brief description of feature]
...
+
+* If you have a running bundle on your environment you can "Duplicate" the bundle and wait for it to build, then "Promote" it to be live - options are under the three vertical dots menu on the deployer page
+* If you do not have a bundle that has the `BLOCK_DIST_TAG` set to `canary` you will need to create a bundle, upload and then "Deploy"
+
+### Creating a feature pack
+
+If you need to create a feature pack to set either `BLOCK_DIST_TAG` to a specific tag, or update blocks you wish to have deployed or change site settings you need to:
+
+1. Have the Fusion-News-Theme repo checked out - https://github.com/WPMedia/Fusion-News-Theme
```
Comment (nschubach) [inline News Theme Development.md:143]:
```suggestion
1. Have the Fusion-News-Theme repo `master` branch checked out - https://github.com/WPMedia/Fusion-News-Theme
```

**Review by nschubach — COMMENTED** [review]
One minor update just for clarity since I asked about it in slack.

**Pull request merged**